### ISSUE TITLE
autolabeler not working

### ISSUE THREAD

**Issue by patriknw** [lightbend/akkaserverless-java-sdk#551]
It's supposed to add labels to the PRs. https://github.com/lightbend/akkaserverless-java-sdk/blob/main/.github/release-drafter.yml#L26

We have some closed PRs with labels but seems like those labels were manually added?

**Comment by ennru** [comment]
Yes, I added labels manually to a few PRs prior to that release as the PRs existed before we merged the release drafter updates. Created #566 now to see if file-based labeling kicks in.

**Comment by ennru** [comment]
"Auto labeler won't work with forks as the token cannot work across repo permissions, and that would be a problem with GitHub permissions." 
https://github.com/release-drafter/release-drafter/issues/785#issuecomment-778700859

**Comment by ennru** [comment]
If we want auto-labelling by file only, we should consider https://github.com/mithro/autolabeler which we use in Alpakka.
